utam-core/src/main/java/utam/core/element/DragAndDropOptions.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.core.element;
+
+import static utam.core.selenium.element.ElementAdapter.ERR_DRAG_AND_DROP_NULL_ELEMENT;
+
+import java.time.Duration;
+import utam.core.framework.UtamCoreError;
+
+/**
+ * options for a drag and drop actions: element, offset and duration.
+ *
+ * @since 236
+ * @author elizaveta.ivanova
+ */
+public interface DragAndDropOptions {
```

Comment:
@jimevanssfdc moved options here to separate location + replaced Point with 2 getters for offset